workday/15.0/modules/ROOT/pages/workday-reference.adoc

```diff
@@ -2706,252 +2754,327 @@ Invoke the Workforce Planning web service, which contains operations that expose
+
+* Configure a larger buffer size to avoid the number of times Mule needs to write the buffer on disk. This increases performance, but it also limits the number of concurrent requests your application can process, because it requires additional memory.
+
+* Configure a smaller buffer size to decrease memory load at the expense of response time. |  |
 | Buffer Unit a| Enumeration, one of:
 
 ** BYTE
 ** KB
 ** MB
-** GB | The unit in which all these attributes are expressed |  |
+** GB | Unit for the *In Memory Size* field. |  |
 |===
 
-[[repeatable-file-store-stream]]
-=== Repeatable File Store Stream
+[[repeatable-in-memory-stream]]
+=== Repeatable In-Memory Stream
 
-File store repeatable streams require buffering, and there are different buffering strategies. Mule keeps a portion of contents in memory. If the stream contents are larger than the configured buffer size, Mule backs up the buffer’s content to disk and then clears the memory.
+Configures the in-memory streaming strategy by which the request fails if the data exceeds the MAX buffer size. Always run performance tests to find the optimal buffer size for your specific use case.
 
-[%header%autowidth.spread]
+[%header,cols="20s,25a,30a,15a,10a"]
 |===
 | Field | Type | Description | Default Value | Required
-| In Memory Size a| Number | Defines the maximum memory that the stream should use to keep data in memory. If more than that is consumed, then it will start to buffer the content on disk. |  |
+| Initial Buffer Size a| Number | Initial amount of memory to allocate to the data stream. If the streamed data exceeds this value, the buffer expands by *Buffer Size Increment*, with an upper limit of *Max In Memory Size* value. |  |
+| Buffer Size Increment a| Number | Amount by which the buffer size expands if it exceeds its initial size. Setting a value of `0` or lower specifies that the buffer can't expand. |  |
+| Max Buffer Size a| Number | Maximum amount of memory to use. If more than that is used then a `STREAM_MAXIMUM_SIZE_EXCEEDED` error is raised. A value lower than or equal to zero means no limit. |  |
 | Buffer Unit a| Enumeration, one of:
 
 ** BYTE
 ** KB
 ** MB
-** GB | The unit in which maxInMemorySize is expressed |  |
+** GB | Unit for the *Initial Buffer Size*, *Buffer Size Increment*, and *Buffer Unit* fields. |  |
 |===
 
-[non-repeatable-stream]
-=== Non-repeatable Stream
+[[SoapAttributes]]
+=== SOAP Attributes
 
-Disable the repeatable stream functionality and use non-repeatable streams to have less performance overhead, memory use, and cost.
+Configures the SOAP attributes type.
 
-[[Default]]
-=== Default
-
-[%header%autowidth.spread]
+[%header,cols="20s,25a,30a,15a,10a"]
 |===
 | Field | Type | Description | Default Value | Required
-| Host a| String |  |  | x
-| Port a| Number |  |  | x
-| Username a| String |  |  |
-| Password a| String |  |  |
-| Non Proxy Hosts a| String |  |  |
+| Transport Additional Data a| Object | Transport additional data. |  |
```

Comment:
The descriptions in this row and the next aren't informative at all. I suggest noting this in a readme for this connector so we can try to get better ones in the future.